backend/src/core/logger.ts

```diff
@@ -29,7 +43,21 @@ class Logger implements LoggerInterface {
       },
     }
   }
+
+  loggerWithLabel(module: NodeModule): any {
+    const label = getModuleLabel(module)
+    return {
+      info: (logMeta: any): winston.Logger =>
+        this.logger.info({ label, ...logMeta }),
+      error: (logMeta: any): winston.Logger =>
+        this.logger.error({ label, ...logMeta }),
+      debug: (logMeta: any): winston.Logger =>
+        this.logger.debug({ label, ...logMeta }),
+      warn: (logMeta: any): winston.Logger =>
+        this.logger.warn({ label, ...logMeta }),
+    }
+  }
 }
 
-const logger = new Logger().logger
+const logger = new Logger()
```

Comment:
Does Logger really need to be a class? It's supposed to be a singleton. So technically, this module could be restructured and exported as
```js
const getModuleLabel = ()=>{}
const logger = winston.createLogger(...)
logger.stream = ...
const loggerWithLabel = () => {}
export {
getStream = () => (logger.stream),
loggerWithLabel
}
```

My aim is to prevent devs from accidentally using logger.logger instead of loggerWithLabel. 

Alternatively, don't change the class, but just export the relevant methods
```js
const logger = new Logger()
export { getStream = () => (logger.stream), loggerWithLabel: (...args) => (logger.loggerWithLabel(...args))}
```